vulture/core.py

```diff
@@ -246,13 +261,13 @@ def report(self):
         for item in self.get_unused_code():
             if self.sort_by_size:
                 line_format = 'line' if item.size == 1 else 'lines'
-                size_report = ' ({0:d} {1})'.format(item.size, line_format)
+                size_report = ' {0:d} {1}'.format(item.size, line_format)
             else:
                 size_report = ''
 
-            print("{0}:{1:d}: {2}{3}".format(
-                utils.format_path(item.filename), item.lineno, item.message,
-                size_report))
+            print("{0}:{1:d}: {2} ({4} % confidence, {3})".format(
```

Comment:
no space between number and %

prepend comma in `size_report` (run locally to check output format)